main/src/cgeo/geocaching/connector/trackable/GeokretyConnector.java

```diff
@@ -191,6 +217,41 @@ public String getTrackableCodeFromUrl(@NonNull final String url) {
     }
 
     @Override
+    @Nullable
+    public String getTrackableTrackingCodeFromUrl(@NonNull final String url) {
+        // http://geokrety.org/m/qr.php?nr=<TRACKING_CODE>
+        final String gkTrackingCode = StringUtils.substringAfterLast(url, "qr.php?nr=");
+        if (StringUtils.isAlphanumeric(gkTrackingCode)) {
+            return gkTrackingCode;
+        }
+        return null;
+    }
+
+    /**
+     * Lookup Trackable Geocode from Tracking Code.
+     *
+     * @param trackingCode
+     *          the Trackable Tracking Code to lookup
+     * @return
+     *          the Trackable Geocode
+     */
+    @Nullable
+    public static String getGeocodeFromTrackingCode(final String trackingCode) {
```

Comment:
IIRC original call comes from https://github.com/kumy/cgeo/blob/issue-5135/main/src/cgeo/geocaching/TrackableActivity.java#L213 which itself is called from `onCreate()`